lib/Service/AvatarService.php

```diff
@@ -93,6 +93,7 @@ private function hasAllowedMime(Avatar $avatar): bool {
 					'image/jpeg',
 					'image/png',
 					'image/x-icon',
+					'image/svg+xml',
```

Comment:
Either we have to support SVG or we need to convert it to PNG format via Imagick (or some other method).

Reply:
I'm not sure if we want to display untrusted SVGs. This is dangerous because SVGs may contain arbitrary javascript code and thus enable XSS attacks.

Reply:
https://github.com/nextcloud/contacts/pull/2407 for a similar situation. There we sanitize the image before it's shown.

Reply:
Okay, so I will convert it into an PNG image. Anyway, BIMI supports Tiny PS, a subset of SVG without support of scripts or animations. Therefore I will check if it's an SVG image, then if it's an Tiny PS image, and if every check was truthy I will convert it to an PNG image. Is there a best practice for the required image size, like 128x128 pixels or 80x80 pixels?